### PR TITLE
Batch of styling and test isolation fixes

### PR DESCRIPTION
**Pull request by invidian**
Some generic fixes taken from #9514, which are useful regardless of enabling the parallel execution of tests.

### CODE REVIEW

File: web/web_test.go [hunk 1]
```diff
@@ -572,33 +574,34 @@ func TestAgentAPIEndPoints(t *testing.T) {
 	webHandler := New(nil, opts)
 	webHandler.Ready()
 
+	baseURL := "http://localhost" + port
```
Comment (roidelapluie) [inline web/web_test.go:577]:
```suggestion
	baseURL := "http://localhost" + port + "/api/v1"
```

Comment (invidian) [reply web/web_test.go:577]: Note, this is about #9711.

BTW, `/api/v1/status` gives 404, `/status` returns 200 when assets are generated.

Also, is `api/v1/-/labels` a valid URI?

Comment (roidelapluie) [reply web/web_test.go:577]: /api/v1/status/runtimeinfo is the one

api/v1/-/labels is not valid. generally we could test that /-/ is not valid.

Comment (invidian) [reply web/web_test.go:577]:
```
    web_test.go:604: http://localhost:33605/api/v1/status/runtimeinfo
    web_test.go:612: 404 page not found
```

Same error for `/api/v1/targets`. Perhaps there is some configuration missing?

>generally we could test that /-/ is not valid.

So with this suggestion, we could either test `/api/v1/-/` or drop the suggesion. 

Even though trying via CLI does work. Can we leave those fixes for separate PR?

**Comment by invidian** [comment]
CI is passing now.

**Review by roidelapluie — APPROVED** [review]

**Comment by roidelapluie** [comment]
Thanks!

**Comment by codesome** [comment]
Please squash and merge :)

**Pull request merged**